pip-selfcheck.json

```diff
@@ -0,0 +1 @@
+{"last_check":"2016-09-26T19:36:56Z","pypi_version":"8.1.2"}
```

Comment:
I added it to the .gitignore as it seems to be generated when I try to run the tests